flux/stdlib/influxdata/influxdb/rules_test.go

```diff
@@ -133,6 +133,7 @@ func TestPushDownRangeRule(t *testing.T) {
 		tc := tc
 		t.Run(tc.Name, func(t *testing.T) {
 			t.Parallel()
+			t.Skip("TODO(dstrand1): Fix to support go-cmp comparisons with protobufs. See in idpe: https://github.com/influxdata/idpe/blob/1326013a0cac99234e947ad5ed02b9fde21db95e/query/stdlib/influxdata/influxdb/rules_test.go#L199")
```

Comment:
this seems like a pretty important test - do we have a plan to unskip it?

Reply:
Maybe we could convert the protos to json and compare that way?

Reply:
IDPE still has these as skipped, I can attempt to convert to json but I'm not sure if it will work properly still. There exists a `protocmp` package that might be useful: https://pkg.go.dev/google.golang.org/protobuf/testing/protocmp but I haven't figured out how to use it yet. @benbjohnson do you have anything to add to this?

Reply:
I hadn't seen the `protocmp` package before. I implemented a `cmp` option for ignoring the unexported fields in `proto.Message` types: https://github.com/influxdata/idpe/blob/387eee6e41d4ccc9883d51e968f6f2fc80beb72e/pkg/cmputil/cmputil.go#L13-L15